src/config/mod.rs

```diff
@@ -202,9 +198,9 @@ fn get_toml_path(dir: &Path) -> Result<Option<PathBuf>, Error> {
     Ok(None)
 }
 
-fn config_path(options: &CliOptions) -> FmtResult<Option<PathBuf>> {
-    let config_path_not_found = |path: &str| -> FmtResult<Option<PathBuf>> {
-        Err(FmtError::from(format!(
+fn config_path(options: &CliOptions) -> Result<Option<PathBuf>, Error> {
+    let config_path_not_found = |path: &str| -> Result<Option<PathBuf>, Error> {
+        Err(Error::new(ErrorKind::NotFound, format!(
             "Error: unable to find a config file for the given path: `{}`",
```

Comment:
Same as above, we can still raise io:error here. It also allows us to give a better context. Converting this error tto a FmtResult later is transparent (handle by the `?` syntax)